spec/sync/storage-manager-listener-spec.js

```diff
@@ -0,0 +1,43 @@
+const getStorageManagerListener = require('./../../lib/sync/storage-manager-listener');
+
+describe('storageManagerListener', () => {
+  it('returns a function', () => {
+    expect(typeof getStorageManagerListener).toEqual('function');
+  });
+
+  describe('when a store is not passed in the first time it is invoked', () => {
+    it('throws an error', () => {
+      expect(() => getStorageManagerListener()).toThrow(new Error('You must pass in a redux store the first time you call getStorageManagerListener'));
+      expect(() => getStorageManagerListener({})).toThrow(new Error('You must pass in a redux store the first time you call getStorageManagerListener'));
```

Comment:
We should split these expectations into two blocks to make it easier to understand which case fails.